src/www/app/fake_net.ts

```diff
@@ -15,7 +15,7 @@
 import {NativeNetworking} from './net';
 
 export class FakeNativeNetworking implements NativeNetworking {
-  async isServerReachable(hostname: string, port: number) {
```

Comment:
This is implementing the interface method. I think we should keep all parameters.